src/functions.js

```diff
@@ -103,16 +103,16 @@ function generateDockerFile(state) {
 
   //TODO bug! space at start
 
-  DockerFile += 'FROM bgruening/galaxy-stable:15.05\n\n';
-  DockerFile += 'MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com\n\n';
-  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\n';
-  DockerFile += 'WORKDIR /galaxy-central\n\n';
+  DockerFile += 'FROM ' + state.baseimage +'\n\r\n\r';
+  DockerFile += 'MAINTAINER Björn A. Grüning, bjoern.gruening@gmail.com\n\r\n\r';
+  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\r\n\r';
+  DockerFile += 'WORKDIR /galaxy-central\n\r\n\r';
```

Comment:
Oh, okay, never tried to build an image in windows, but if it is not possible than the ‘/r’ are not necessary. :)

 

 

Van: Björn Grüning [mailto:notifications@github.com] 
Verzonden: 08 August 2015 16:40
Aan: galaxyFlavorsGenerators/galaxyFlavorGenerator <galaxyFlavorGenerator@noreply.github.com>
CC: Jeroen Galle <jeroen.galle@ugent.be>
Onderwerp: Re: [galaxyFlavorGenerator] added base image select (#16)

 

In src/functions.js <https://github.com/galaxyFlavorsGenerators/galaxyFlavorGenerator/pull/16#discussion_r36578863> :

> @@ -103,16 +103,16 @@ function generateDockerFile(state) {
>  
>    //TODO bug! space at start
>  
> -  DockerFile += 'FROM bgruening/galaxy-stable:15.05\n\n';
> -  DockerFile += 'MAINTAINER Björn A. Grüning,  <mailto:bjoern.gruening@gmail.com\n\n> bjoern.gruening@gmail.com\n\n';
> -  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\n';
> -  DockerFile += 'WORKDIR /galaxy-central\n\n';
> +  DockerFile += 'FROM ' + state.baseimage +'\n\r\n\r';
> +  DockerFile += 'MAINTAINER Björn A. Grüning,  <mailto:bjoern.gruening@gmail.com\n\r\n\r> bjoern.gruening@gmail.com\n\r\n\r';
> +  DockerFile += 'ENV GALAXY_CONFIG_BRAND "' + state.GALAXY_CONFIG_BRAND + '"\n\r\n\r';
> +  DockerFile += 'WORKDIR /galaxy-central\n\r\n\r';

@jegalle <https://github.com/jegalle>  are these line-breaks really needed? I don't think it's possible to build images on Windows.

—
Reply to this email directly or view it on GitHub <https://github.com/galaxyFlavorsGenerators/galaxyFlavorGenerator/pull/16/files#r36578863> .  <https://github.com/notifications/beacon/AL9cSsGugbSABstEW_I40RIXnv6B49Ifks5olgxNgaJpZM4FoBIl.gif>